android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,289 @@
+            )
+
+            FavoriteButton(
+                modifier = Modifier
+                    .padding(
+                        start = MaterialTheme.spacing.medium,
+                        bottom = MaterialTheme.spacing.medium
+                    )
+                    .size(30.dp),
+                isFavourite = isFavorite,
+                onClick = { }
+            )
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsOverview(
+    modifier: Modifier = Modifier,
+    overview: String
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        MovieDetailsSectionHeader(
+            textId = R.string.overview
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        Text(
+            modifier = Modifier
+                .fillMaxWidth(),
+            text = overview,
+            maxLines = 7,
+            fontSize = 14.sp
+        )
+    }
+}
+
+@Composable
+fun MovieDetailsCrew(
+    modifier: Modifier = Modifier,
+    crew: List<CrewmanViewState>
+) {
+    Column(
+        modifier = modifier
+            .heightIn(min = 50.dp, max = 400.dp)
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        LazyVerticalGrid(
+            columns = GridCells.Fixed(3),
+            verticalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            userScrollEnabled = false
+        ) {
+            items(
+                items = crew,
+                key = { crewman -> crewman.id }
+            ) { crewman ->
+                CrewItem(
+                    item = CrewItemViewState(
+                        crewman.name,
+                        crewman.job
+                    )
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsCast(
+    modifier: Modifier = Modifier,
+    cast: List<ActorCardViewState>
+) {
+    Column(
+        modifier = modifier
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        MovieDetailsSectionHeader(
+            textId = R.string.top_billed_cast
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        LazyRow(
+            horizontalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            userScrollEnabled = true
+        ) {
+            items(
+                items = cast,
+                key = { actor -> actor.id }
+            ) { actor ->
+                ActorCard(
+                    item = ActorCardViewState(
+                        actor.id,
+                        actor.name,
+                        actor.character,
+                        actor.imageUrl
+                    )
```

Comment:
`item = actor`

Reply:
@ivanseki Please fix this

Reply:
@ivan-varga I fixed that already

Reply:
Seems like github had some issues and I weren't able to see your changes. It's fixed now and going to be approved ASAP